nextcord/utils.py

```diff
@@ -1131,6 +1131,30 @@ def format_dt(dt: datetime.datetime, /, style: Optional[TimestampStyle] = None)
     return f"<t:{int(dt.timestamp())}:{style}>"
 
 
+def format_as_channel_name(s: str, include_hashtag: bool = False) -> str:
```

Comment:
```suggestion
def format_as_channel_name(string: str, *, include_hashtag: bool = False) -> str:
```
Just to be more descriptive.

Reply:
Why was this resolved without any conversation?

Reply:
+1, I don't see a reason for these to be resolved without any conversation or explanation

Reply:
Just an extension to this, one character variable names are awful for readability, and the key word argument being forced is because `format_as_channel_name("name", True)` has no semantic meaning for `True`, but `format_as_channel_name("name", include_hashtag=True)` does.